### ISSUE TITLE
iOS16.2 多巴胺rootless越狱无法正常使用

### ISSUE THREAD

**Issue by sublimter** [masterKing/RevealLoader2#1]
已安装revealloader2，安装了version 46版本的reveal，iphone设置里面打开了相关app，再打开对应的app；最后打开reveal，无法显示监听的app UI

**Comment by masterKing** [comment]
可以打开 Mac 上的控制台，左侧选择你的设备，右上角输入你要调试的 APP 进程名过滤其他不需要看的信息。然后打开 App 看看 RevealLoader2.dylib 的打印信息，如果无法显示应该会有 dlopen error: ... 的错误日志，把这个日志贴出来看看

**Comment by imkh** [comment]
I have the same issue (iPhone 12 on iOS 16.1.2 rootless jailbreak with Dopamine). I looked at the Console.app logs and it seems to be because the RevealServer binary is `arm64` when it needs `arm64e`. Is there an `arm64e` Reveal Server binary? I can't see one on my Mac with the latest version of Reveal (v52).

```
dlopen error: dlopen(/private/preboot/[...]/dopamine-bFyI7F/procursus/Library/Frameworks/RevealServer.framework/RevealServer, 0x0002): tried: '/private/preboot/[...]/dopamine-bFyI7F/procursus/Library/Frameworks/RevealServer.framework/RevealServer' (mach-o file, but is an incompatible architecture (have 'arm64', need 'arm64e')), '/private/preboot/Cryptexes/OS/private/preboot/[...]/dopamine-bFyI7F/procursus/Library/Frameworks/RevealServer.framework/RevealServer' (errno=2), '/private/preboot/[...]/dopamine-bFyI7F/procursus/Library/Frameworks/RevealServer.framework/RevealServer' (mach-o file, but is an incompatible architecture (have 'arm64', need 'arm64e'))
```

![Image](https://github.com/user-attachments/assets/a8187e79-20bf-47bc-b1ac-9412a40a45d9)

EDIT: Apparently, they're not shipping `arm64e` slices anymore since 2022, according to this [tweet](https://x.com/reveal_app/status/1547376050594025472).

![Image](https://github.com/user-attachments/assets/b8b99fa3-6d4e-46a3-9b5a-4206d7bb8c8a)

**Comment by oahgnehzoul** [comment]
dlopen error: dlopen(/private/preboot/CFBE6BE078C70AD4983FF7774D629E73EF03AED3/dopamine-Kp39rn/procursus/Library/Frameworks/RevealServer.framework/RevealServer, 0x0002): tried: '/private/preboot/CFBE6BE078C70AD4983FF7774D629E73EF03AED3/dopamine-Kp39rn/procursus/Library/Frameworks/RevealServer.framework/RevealServer' (code signature invalid (errno=1) sliceOffset=0x00000000, codeBlobOffset=0x004C9AA0, codeBlobSize=0x0000E160 for '/private/preboot/CFBE6BE078C70AD4983FF7774D629E73EF03AED3/dopamine-Kp39rn/procursus/Library/Frameworks/RevealServer.framework/RevealServer'), '/System/Library/Frameworks/RevealServer.framework/RevealServer' (no such file)   帮看下我这个报错咋解决？

**Comment by w2599** [comment]
Yes, it is currently found that only arm64 apps are supported, such as SpringBoard and Preference, which are all arm64e.
